cli/pkg/kctrl/cmd/core/secure_namespace_flags.go

```diff
@@ -18,7 +18,7 @@ type SecureNamespaceFlags struct {
 	AllowedSharedNamespaces bool
 }
 
-func (s *SecureNamespaceFlags) Set(cmd *cobra.Command) {
+func (s *SecureNamespaceFlags) Set(cmd *cobra.Command, defaultVal bool) {
 	cmd.Flags().BoolVar(&s.AllowedSharedNamespaces, "dangerous-allow-use-of-shared-namespace", false, "Allow use of shared namespaces")
```

Comment:
defaultVal should be used, right?
```suggestion
	cmd.Flags().BoolVar(&s.AllowedSharedNamespaces, "dangerous-allow-use-of-shared-namespace", defaultVal, "Allow use of shared namespaces")
```

Reply:
Yep, will make this change